src/regions.jl

```diff
@@ -0,0 +1,163 @@
+"""
+     function surfaceregion(TMIversion::String,region::String,γ::Grid)::BoundaryCondition
+
+    Read an oceanographically-relevant surface region from NetCDF file. (Also could be read from mat file.)
+    Return a BoundaryCondition
+"""
+function surfaceregion(TMIversion::String,region::Union{String,Symbol})::BoundaryCondition
+
+    file,Nx,Ny = download_regionfile(TMIversion::String)
+    
+    # version 1: region masks were stored with "d" prefix
+    # new version: regions defined by region name alone
+    tracername = Symbol(region)
+
+    #dsfc = ncread(file,tracername) # using NetCDF.jl
+    ds = Dataset(file,"r") # using NCDatasets.jl
+    v = ds[tracername]
+    units = v.attrib["units"]
+    longname = v.attrib["longname"]
+
+    lon = ds["lon"][:]
+    lat = ds["lat"][:]
+    depth = ds["depth"][1]
+    mask = Bool.(v[:,:]) # use BitMatrix to save some bits
+
+    #name = v.attrib["name"] # error
+    close(ds)
+    
+    b = BoundaryCondition(mask,lon,lat,depth,3,1,trues(parse(Int64,Nx),parse(Int64,Ny)),Symbol(region),longname,units)
```

Comment:
Why is the `wet` set to a Matrix of `true`s?

Reply:
yes, it could be set to the land mask. That would help with visualization via UnicodePlots. It wouldn't change any numerical calculations, however, and it would require a new argument to the BoundaryCondition constructor. Do you think it is worth it?

Reply:
Probably worth it? Feels like if you are able to call `BoundaryCondition.wet`, it should be returning the correct value. I feel like the easiest way to do it would be to add a new field to the `region_90x45.nc` and `region_180x90.nc` files that's `wet` and just load that in in this function. I can probably make those changes while I'm updating the regions if that sounds like an appropriate approach.

Reply:
I think it would be bug-prone to read in two potentially inconsistent wet masks. If you wanted to change the function arguments to grab the already-existing wet mask, that would be the better approach and would be appreciated.

Reply:
sure, can do.